packages/lib/src/components/internal/SecuredFields/binLookup/triggerBinLookUp.ts

```diff
@@ -4,7 +4,6 @@ import { DEFAULT_CARD_GROUP_TYPES } from '../lib/configuration/constants';
 import { getError } from '../../../../core/Errors/utils';
 import { ERROR_MSG_UNSUPPORTED_CARD_ENTERED } from '../../../../core/Errors/constants';
 import { BinLookupResponse, BinLookupResponseRaw } from '../../../Card/types';
-import { sortBrandsAccordingToRules } from './sortBinLookupBrands';
```

Comment:
This function can be deleted, right?

Reply:
I'm keeping it there for the time being until the backend changes have been fully confirmed to work (no merchant complaints!)